test/extended/testdata/builds/volumes/csi-docker-buildconfig.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: build.openshift.io/v1
+kind: BuildConfig
+metadata:
+  name: mydockertest
+  labels:
+    name: test
+spec:
+  output:
+    to:
+      kind: ImageStreamTag
+      name: mydockerstream:latest
+  postCommit:
+    script: cat /var/run/secrets/some-secret/key
+  triggers: []
+  runPolicy: Serial
+  source:
+    dockerfile:
+      'FROM registry.ci.openshift.org/ocp/4.8:test-build-simples2i'
```

Comment:
Please note that this image is being relocated for multi-arch support in #26598

Reply:
ack @yselkowitz